internal/cmd/server.go

```diff
@@ -57,9 +57,11 @@ func server() error {
 		return err
 	}
 
-	config := spew.NewDefaultConfig()
-	config.DisableMethods = true
-	config.Dump(cfg)
+	if cfg.EnvoyGateway.Admin.Debug {
```

Comment:
```suggestion
	if cfg.EnvoyGateway && cfg.EnvoyGateway.Admin && cfg.EnvoyGateway.Admin.Debug {
```

Reply:
`Admin` has been initialized in API helper. maybe just init it when configpath="" and Admin is nil

Reply:
Is that's the case then this is fine